fling/component/client/client_template.py

```diff
@@ -148,3 +148,147 @@ def test(self) -> dict:
             - A diction containing testing results.
         """
         raise NotImplementedError
+
+class CrossDomainClientTemplate:
```

Comment:
I suppose that the difference between this class and ``ClientTemplate`` only lies in the construction function. So, you can simply inherit ``ClientTemplate`` with minor modifications.